local/query_opensrc.py

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
We shouldn't have a separate query controller for opensource - it should be an argument in the answer function